src/index.ts

```diff
@@ -45,6 +45,42 @@ function toTypeScriptType(property: XtpTyped): string {
   return toTypeScriptTypeX(property.xtpType!)
 }
 
+function getDefault(property: XtpTyped): string {
+  return getDefaultForType(property.xtpType!)
+}
+
+function getDefaultForType(type: XtpNormalizedType): string {
```

Comment:
i'm kind of confused what's happening here. why does it need a default type? javascript has nulls